api/app/src/main/kotlin/packit/controllers/PacketGroupController.kt

```diff
@@ -30,12 +32,33 @@ class PacketGroupController(
         return ResponseEntity.ok(packetGroupService.getPacketGroups(payload, filterName).map { it.toDto() })
     }
 
-    @GetMapping("/{name}/display")
+    @GetMapping("/packetGroups/{name}/display")
     fun getDisplay(
         @PathVariable name: String
     ): ResponseEntity<PacketGroupDisplay> {
         val result = packetGroupService.getPacketGroupDisplay(name)
 
         return ResponseEntity.ok(result)
     }
+
+    @GetMapping("/packetGroups/{name}/packets")
+    fun getPackets(
+        @PathVariable name: String,
+    ): ResponseEntity<List<PacketDto>>
+    {
+        return ResponseEntity.ok(
+            packetService.getPacketsByName(name).map { it.toDto() }
+        )
+    }
+
+    @GetMapping("/packetGroupSummaries")
```

Comment:
You'd rather have this than `/packetGroups/summaries`? At least then everything in this controller would have the same base route. I don't think that would be ambiguous as no other routes have two parts under `packetGroups`.

Reply:
Yes, even if it isn't technically ambiguous today, conceptually I think it is cleaner if we reserve the `/packetGroups` route to only have packet names as the next segment. It seems likely that we'll have a `/packetGroups/<name>` in the near future (eg. by renaming the `/display` route) or maybe even a `/packetGroupSummaries/<name>`, in which case there will be an actual ambiguity.